src/app/features/registration/registration.component.ts

```diff
@@ -1,10 +1,53 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, Output } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { faEye, faEyeSlash } from '@fortawesome/free-solid-svg-icons';
+import { APP_ROUTS } from 'src/app/app.model';
 
 @Component({
   selector: 'app-registration',
   templateUrl: './registration.component.html',
   styleUrls: ['./registration.component.scss'],
 })
 export class RegistrationComponent {
-  constructor() {}
+  hidePassword = true;
```

Comment:
Look at the comments on the login component and correct anything you can here.

Reply:
fixed